public/js/apps/management/app.jsx

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+var React = require('react');
+var {
+  Accordion,
+  AccordionContent,
+  AccordionSection} = require('components/accordion');
+
+var gettext = require('utils').gettext;
+
+
+var Management = React.createClass({
+
+  displayName: 'ManagementApp',
+
+  render: function() {
+    return (
+
+      <div>
+        <header className="top-nav">
+          <h1 className="logo">Firefox Payments</h1>
+          <button>{gettext('Sign Out')}</button>
+        </header>
+
+        <div className="content">
+          <div className="user">
+            <p>Hello, placeholder@placeholder.com</p>
```

Comment:
this needs localization. We could fix it up later when we add a real email though.